components/director/internal/domain/labeldef/repository_test.go

```diff
@@ -100,6 +100,87 @@ func TestRepositoryCreateLabelDefinition(t *testing.T) {
 	})
 }
 
+func TestRepositoryUpdateLabelDefinition(t *testing.T) {
+	// GIVEN
+	labelDefID := "d048f47b-b700-49ed-913d-180c3748164b"
+	tenantID := "003a0855-4eb0-486d-8fc6-3ab2f2312ca0"
+	someString := "any"
+	var someSchema interface{}
+	someSchema = someString
+
+	in := model.LabelDefinition{
+		ID:     labelDefID,
+		Tenant: tenantID,
+		Key:    "some-key",
+		Schema: &someSchema,
+	}
+
+	t.Run("successfully updated definition with schema", func(t *testing.T) {
+		db, dbMock := mockDatabase(t)
```

Comment:
Formatting, additional new lines would make it more readable.

Reply:
done